lib/improver/tests/weights/test_ChooseDefaultWeightsTriangular.py

```diff
@@ -60,19 +60,19 @@ def test_basic(self):
         """Test that the function returns a numpy array.
            Also check that the length of the weights is correct and they add
            up to 1.0"""
-        TriangularWeightsClass = ChooseDefaultWeightsTriangular(3, 5)
+        TriangularWeightsClass = ChooseDefaultWeightsTriangular(3)
         coord_vals = np.arange(15)
-        weights = TriangularWeightsClass.triangular_weights(coord_vals)
+        weights = TriangularWeightsClass.triangular_weights(coord_vals, 5)
```

Comment:
Generally, I think it's preferable to define `midpoint = 5` and pass midpoint into the `triangular_weights` function, so that it's clear what this function is referring to, from the unit test.